circle.yml

```diff
@@ -7,40 +7,41 @@ machine:
   hosts:
     testpilot.dev: 127.0.0.1
   node:
-    version: 6.2.0
+    version: 6.9.1
   environment:
     TESTPILOT_ENABLE_PONTOON_BRANCHES: master
 
 dependencies:
 
   cache_directories:
     - "addon/node_modules"
-    - "node_modules"
-
-  pre:
-    # https://discuss.circleci.com/t/nvm-commands-are-not-evaluated/943
-    - nvm use 6.2.0 && nvm alias default 6.2.0
+    # node_modules is included by default
 
   override:
-    - node --version
-    - npm --version
+    - npm install
+    - cd addon && npm install
     - google-chrome --version
     - ./bin/circleci/setup-test-dependencies.sh
-    - ./bin/circleci/build-version-json.sh
     - ./bin/circleci/build-addon.sh
-    - ./bin/circleci/build-frontend.sh
 
 test:
   override:
+    - ./bin/circleci/test-addon.sh
```

Comment:
:+1: on separating test from build. I started to do that at some point with the Django cleanup but got distracted by something shiny and wandered off